site/docs/updating-bazel.md

```diff
@@ -5,11 +5,21 @@ title: Updating Bazel
 
 # Updating Bazel
 
-Bazel defined a [backwards compatibility policy](https://docs.bazel.build/versions/master/backward-compatibility.html)
+Bazel has a [backwards compatibility policy](https://docs.bazel.build/versions/master/backward-compatibility.html)
```

Comment:
Question to think about.   Should we use 'Bazel' to mean bazel the code or in this broader sense of 'the bazel project'.
Or should we have distinct words to disambiguate.

Reply:
+1 "The Bazel maintainers have a.."

Reply:
Done.